change/@ni-nimble-components-897a77b2-4280-475f-977b-21a0571b7acf.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This could be a good topic for our next call with our Chromatic rep. If you agree, it'd be good to save off some links/screenshots/chromatic builds/notes to show why we didn't end up using the icon gallery doc block.